checkov/terraform/checks/resource/base_resource_negative_value_check.py

```diff
@@ -0,0 +1,38 @@
+from abc import abstractmethod
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class BaseResourceNegativeValueCheck(BaseResourceCheck):
+    def __init__(self, name, id, categories, supported_resources):
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf):
```

Comment:
@rotemavni This method should be implemented here, for all the other checks

Reply:
@nimrodkor implemented the method